pages/admin/index.tsx

```diff
@@ -11,80 +11,149 @@ import {
 import Table from '@components/internal/Table'; // Table component
 import Pagination from '@components/internal/Pagination'; // Pagination component
+import { RequestStatusBadge } from '@components/internal/RequestStatusBadge'; //Status badge component
+
+type StatusProps = {
+  value: any;
+};
+
+function renderStatusBadge({ value }: StatusProps) {
+  return (
+    <Wrap>
+      <RequestStatusBadge variant={value}></RequestStatusBadge>
+    </Wrap>
+  );
+}
+
+type NameProps = {
+  readonly value: any;
+};
+
+function renderName({ value }: NameProps) {
+  return (
+    <div>
+      <Text>{value.name}</Text>
+      <Text textStyle="caption" textColor="secondary">
+        ID: {value.id}
+      </Text>
+    </div>
+  );
+}
 
 // Placeholder columns
 // TODO: accessors should be the accessors for these fields in the DB
 const COLUMNS = [
   {
     Header: 'Name',
     accessor: 'name',
+    Cell: renderName,
+    minWidth: 240,
+    width: 280,
   },
   {
     Header: 'Date Received',
     accessor: 'dateReceived',
+    maxWidth: 240,
+    width: 240,
   },
   {
     Header: 'Permit Type',
     accessor: 'permitType',
     disableSortBy: true,
+    maxWidth: 180,
+    width: 180,
   },
   {
     Header: 'Request Type',
     accessor: 'requestType',
     disableSortBy: true,
+    maxWidth: 180,
+    width: 180,
   },
   {
     Header: 'Status',
     accessor: 'status',
     disableSortBy: true,
+    Cell: renderStatusBadge,
+    maxWidth: 180,
+    width: 100,
   },
 ];
 
 // Placeholder data
 const DATA = [
   {
-    name: 'Steve Rogers',
+    name: {
+      name: 'Steve Rogers',
+      id: '36565',
+    },
     dateReceived: 'Dec 21 2021, 8:30 pm',
     permitType: 'Permanent',
     requestType: 'Replacement',
-    status: 'In progress',
+    status: 'pending',
   },
   {
-    name: 'Steve Rogers',
+    name: {
+      name: 'Steve Rogers',
+      id: '36565',
+    },
     dateReceived: 'Dec 21 2021, 8:30 pm',
     permitType: 'Permanent',
     requestType: 'Replacement',
-    status: 'In progress',
+    status: 'inProgress',
   },
   {
-    name: 'Steve Rogers',
+    name: {
+      name: 'Steve Rogers',
+      id: '36565',
+    },
     dateReceived: 'Dec 21 2021, 8:30 pm',
     permitType: 'Permanent',
     requestType: 'Replacement',
-    status: 'In progress',
+    status: 'completed',
   },
 ];
 
 // Internal home page - view APP requests
 export default function Requests() {
+  const [permitTypeFilter, setPermitTypeFilter] = useControllableState({ defaultValue: 'All' });
+  const [requestTypeFilter, setRequestTypeFilter] = useControllableState({ defaultValue: 'All' });
+
   return (
     <Layout>
       <GridItem colSpan={12}>
         <Flex justifyContent="space-between" alignItems="center" marginBottom="32px">
           <Text textStyle="display-xlarge">Requests</Text>
-          <Button leftIcon={<AddIcon />} height="48px" padding="24px">
-            Create a request
-          </Button>
+          {/* TODO
```

Comment:
can you add something about the create function being addressed post mvp. It took me a minute to remember why this was commented out, so i can only imagine what its going too be like looking at this again a couple of months 😅